### ISSUE TITLE
(DinoMod) When riding a dinosaur player's and dino's sprites look wrong.

### ISSUE THREAD

**Issue by Dracque** [cataclysmbnteam/Cataclysm-BN#4659]
### Describe the bug

Riding a dinosaur results in the player character being behind the dinosaur instead of on top.
Don't know if it's an issue with all or most dinos.

### Steps To Reproduce

1. Create a character in a world with DinoMod enabled.
2. Find a non hostile dinosaur.
3. Tame it (cattle fodder or bird food work on most).
4. Interact with friendly dinosaur and select "Mount" (may require horse tack if survival is low).

### Screenshots

Character's sprite hidden behind dino's.
![Debug Dan _2024-05-16_18-56-27_+0100](https://github.com/cataclysmbnteam/Cataclysm-BN/assets/167656683/bec0f969-f787-44f2-975c-ca89090f7722)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.22631.3593 (23H2)
- Game Version: ecb842b [64-bit]
- Graphics Version: Tiles
- LAPI Version: 2
- Game Language: English [en_US]
- Mods loaded: [
    Bright Nights [bn],
    Disable NPC Needs [no_npc_food],
    Limit Fungal Growth [limit_fungal_growth],
    DinoMod [DinoMod],
    No Rail Stations [No_Rail_Stations]
]

**Comment by LyleSY** [comment]
Yes, this is because the tileset does not yet have a dedicated riding sprite for this dino. The fix is to create that art

**Comment by Dracque** [comment]
Best I could do would be to make a list of missing sprites since I lack the skills to create new art for the game.

**Comment by Dracque** [comment]
My apologies for the delay.
Below are the Dinos with missing sprites I've found with some additional notes. Included a .txt file version for convenience.
[PET_MOUNTABLE_NoSprite.txt](https://github.com/user-attachments/files/18307701/PET_MOUNTABLE_NoSprite.txt)

#Default tile-set
#Made by searching for "PET_MOUNTABLE" in dinosaur.json, dinosaur_CBM.json, hatchling.json and juvenile.json.
#Found some zdinos with the flag in zed-dinosaur.json, zinosaur_burned.json and zinosaur_upgrade.json but no clue how to tame those.

* must tame juvenile
^#Don't know how to quickly age mobs so couldn't test the above.
! PET_MOUNTABLE not mountable
? not tameable
+ can be tamed

anzu*
sarahsaurus*
anchisaurus*
gallimimus+
nothronychus+
sarahsaurus juvenile!+
^#Spawned all "brown and magenta sauropod juvenile" and couldn't mount.
dryosaurus+!
^#Player too heavy even with Tiny + Hollow Bones mutations and naked (no bionics)
camptosaurus+
stegoceras+!
^#Player too heavy even with Tiny + Hollow Bones mutations and naked (no bionics)
thescelosaurus+
^#Requires at least Little mutation
quetzalcoatlus+
^#Too heavy at normal player size even with Hollow Bones mutation, must be at least Little
issi juvenile+!
^#Spawned all "brown and magenta sauropod juvenile" and couldn't mount.
tenontosaurus juvenile+
einiosaurus juvenile+
kosmoceratops juvenile+
^#Shares name with its adult form

**Comment by Dracque** [comment]
I forgot I could edit the json file to make the first 3 tameable to test for missing sprite.
Added "petfood": { "food": [ "DINOFOOD_C" ] }, saved file and launched the game.
All three, anzu, sarahsaurus and anchisaurus are missing a riding sprite.

Attached updated txt file.
[PET_MOUNTABLE_NoSprite.txt](https://github.com/user-attachments/files/18351900/PET_MOUNTABLE_NoSprite.txt)

Content of said file below if someone would rather avoid downloading.
#Default tile-set
#Made by searching for "PET_MOUNTABLE" in dinosaur.json, dinosaur_CBM.json, hatchling.json and juvenile.json.
#Found some zdinos with the flag in zed-dinosaur.json, zinosaur_burned.json and zinosaur_upgrade.json but no clue how to tame those.

* must tame juvenile
^#Tested by editing json file and adding "petfood": { "food": [ "DINOFOOD_C" ] } to each monster
! PET_MOUNTABLE not mountable
? not tameable
+ can be tamed

anzu*
^#Requires at least Little mutation
sarahsaurus*
^#Requires at least Little mutation
anchisaurus*
^#Player too heavy even with Tiny + Hollow Bones mutations and naked (no bionics)
gallimimus+
nothronychus+
sarahsaurus juvenile!+
^#Spawned all "brown and magenta sauropod juvenile" and couldn't mount.
dryosaurus+!
^#Player too heavy even with Tiny + Hollow Bones mutations and naked (no bionics)
camptosaurus+
stegoceras+!
^#Player too heavy even with Tiny + Hollow Bones mutations and naked (no bionics)
thescelosaurus+
^#Requires at least Little mutation
quetzalcoatlus+
^#Too heavy at normal player size even with Hollow Bones mutation, must be at least Little
issi juvenile+!
^#Spawned all "brown and magenta sauropod juvenile" and couldn't mount.
tenontosaurus juvenile+
einiosaurus juvenile+
kosmoceratops juvenile+
^#Shares name with its adult form
